apps/vela-ext/tests/background.test.ts

```
@@ -0,0 +1,51 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
```

ESLint

[error] 'beforeEach' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)